specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json

```
@@ -0,0 +1,6113 @@
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/RelationshipId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceRelationship"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Devices_CreateRelationship",
+        "summary": "Given the ID for a device and a relationship ID associated with this device, create a new relationship for between the given device and a second device specified in the body.",
+        "description": "Create a device relationship",
+        "x-ms-examples": {
+          "Create a new device relationship": {
+            "$ref": "./examples/devices_relationships_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/RelationshipId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device relationship body.",
+            "schema": {
+              "$ref": "#/definitions/DeviceRelationship"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceRelationship"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Devices_UpdateRelationship",
+        "summary": "Patch a given relationship given the relationship ID and a given device ID.",
+        "description": "Update device relationship",
+        "x-ms-examples": {
+          "Update a device relationship": {
+            "$ref": "./examples/devices_relationships_update.json"
+          }
+        },
+        "consumes": [
+          "application/json",
+          "application/merge-patch+json"
```

Patch should only accept `application/merge-patch+json`
```suggestion
          "application/merge-patch+json"
```

---

Fixed